test/string.spec.ts

```diff
@@ -0,0 +1,483 @@
+import { describe, expect, it } from 'vitest';
+import { faker, FakerError } from '../src';
+import { seededTests } from './support/seededRuns';
+import { times } from './support/times';
+
+const NON_SEEDED_BASED_RUN = 5;
+
+describe('string', () => {
+  seededTests(faker, 'string', (t) => {
+    t.describe('alpha', (t) => {
+      t.it('noArgs')
+        .itRepeated('with length parameter', 5, 5)
+        .it('with length', { length: 6 })
+        .it('with casing = lower', { casing: 'lower' })
+        .it('with casing = upper', { casing: 'upper' })
+        .it('with casing = mixed', { casing: 'mixed' })
+        .it('with bannedChars', { bannedChars: 'abcdefghijk' })
+        .it('with length, casing and bannedChars', {
+          length: 7,
+          casing: 'lower',
+          bannedChars: 'lmnopqrstu',
+        });
+    });
+
+    t.describe('alphanumeric', (t) => {
+      t.it('noArgs')
+        .itRepeated('with length parameter', 5, 5)
+        .it('with length', { length: 6 })
+        .it('with casing = lower', { casing: 'lower' })
+        .it('with casing = upper', { casing: 'upper' })
+        .it('with casing = mixed', { casing: 'mixed' })
+        .it('with bannedChars', { bannedChars: 'abcdefghijk12345' })
+        .it('with length, casing and bannedChars', {
+          length: 7,
+          casing: 'lower',
+          bannedChars: 'lmnopqrstu67890',
+        });
+    });
+
+    t.describe('hexadecimal', (t) => {
+      t.it('noArgs')
+        .it('with length', { length: 6 })
+        .it('with casing = lower', { casing: 'lower' })
+        .it('with casing = upper', { casing: 'upper' })
+        .it('with casing = mixed', { casing: 'mixed' })
+        .it('with custom prefix', { prefix: 'hex_' })
+        .it('with length, casing and empty prefix', {
+          length: 7,
+          casing: 'lower',
+          prefix: '',
+        });
+    });
+
+    t.describe('numeric', (t) => {
+      t.it('noArgs')
+        .itRepeated('with length parameter', 5, 5)
+        .it('with length', { length: 6 })
+        .it('with allowLeadingZeros', { allowLeadingZeros: true })
+        .it('with bannedChars', { bannedDigits: '12345' })
+        .it('with length, allowLeadingZeros and bannedChars', {
+          length: 7,
+          allowLeadingZeros: true,
+          bannedDigits: '12345',
+        });
```

Comment:
```suggestion
        .it('with bannedDigits', { bannedDigits: '12345' })
        .it('with length, allowLeadingZeros and bannedDigits', {
          length: 7,
          allowLeadingZeros: true,
          bannedDigits: '12345',
        });
```
Should this be `bannedDigits` because the parameter is called `bannedDigits`, or are we using `bannedChars` for everything?

Reply:
You are right, was a copy paste issue